package.json

```diff
@@ -99,7 +99,8 @@
       "protobufjs@>=6.10.0 <7.2.4": ">=7.2.4",
       "tough-cookie@<4.1.3": ">=4.1.3",
       "@ethereumjs/util@<9.0.0": ">=9.0.0",
-      "axios@<0.28.0": ">=0.28.0"
+      "axios@<0.28.0": ">=0.28.0",
+      "isows@<1.0.4": ">=1.0.4"
```

Comment:
did this actually end up working?

Reply:
hm no it didn't, need to try a few things here. i guess it only forces the dep for us but not in the bundle

Reply:
yeah so then lets remove it, I think the viem latest version update fixes it then

Reply:
done